hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/contract/ContractCallTransitionLogic.java

```diff
@@ -134,6 +146,13 @@ public void doStateTransitionOperation(
                     ? new Account(targetId)
                     : accountStore.loadContract(targetId);
         }
+        // Since contracts cannot have receiverSigRequired=true, this can only restrict us from sending value to an EOA
+        if (!isMissing && op.getAmount() > 0) {
+            // Use Address.ZERO as "active contract"; will never activate a valid Key{contractID} in the receiver key
+            final var sigReqIsMet = sigsVerifier.hasActiveKeyOrNoReceiverSigReq(
+                    false, target.toEvmAddress(), Address.ZERO, worldLedgers, ContractCall);
```

Comment:
for 2nd argument here - the target is missing (by definition, at this line of code) - so maybe just use a (named) constant value that shows this?

Reply:
Renamed to `NEVER_ACTIVE_CONTRACT_ADDRESS`.

Reply:
Ah, `target` is _not_ missing at this line.